packages/ui/src/components/wui-image/styles.ts

```diff
@@ -3,8 +3,8 @@ import { css } from 'lit'
 export default css`
   :host {
     display: block;
-    width: 100%;
-    height: 100%;
+    width: var(--local-width);
+    height: var(--local-height);
```

Comment:
just checking this is controlled by ` wui-image` directly and it doesn't break other things?

Reply:
Exactly, all seem good. Thats good point that it might affect thousands of UIs :)